modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc

```diff
@@ -24,8 +24,9 @@ spec:
 ----
 <1> For example: `openVSXURL: "pass:c,a,q[https://open-vsx.org]"`.
 +
-[TIP]
-==== 
+[IMPORTANT]
+====
+* Using link:https://open-vsx.org[https://open-vsx.org] with thousands of community-supported extensions is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```

Comment:
```suggestion
* Using the extensions from link:https://open-vsx.org[https://open-vsx.org] is not recommended in an air-gapped environment, isolated from the internet. In order to reduce the risk of malware infections and unauthorized access to your code use the embedded or self-hosted Open VSX registry with the curated set of extensions.
```

Reply:
tbh, I like the original version more

Reply:
potentially, we can remove `with thousands of community-supported extensions` part, but the main point here  is about registry itself (`https://open-vsx.org` in the CR), not individual extentions